app/core/routes.py

```
@@ -655,49 +696,54 @@ def submit_feedback():
-                payload={'feedback_type': feedback_type, 'rating': rating}
+                event_type="feedback_submitted",
+                triggers={"source": "web_ui", "action": "modal_form"},
+                payload={"feedback_type": feedback_type, "rating": rating},
             )
         except Exception:
             pass  # Telemetry failures must not block user flow
 
-        return jsonify({'success': True, 'message': 'Feedback submitted successfully'})
+        return jsonify({"success": True, "message": "Feedback submitted successfully"})
 
     except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)